applications/FluidDynamicsApplication/python_scripts/stokes_solver.py

```diff
@@ -54,13 +54,12 @@ def __init__(self, model_part, settings):
         self.compute_reactions = settings.compute_reactions
         self.reform_dofs_at_each_step = settings.reform_dofs_at_each_step
 
-        import KratosMultiphysics.python_linear_solver_factory as linear_solver_factory
         self.linear_solver = linear_solver_factory.ConstructSolver(settings.linear_solver_settings)
 
         self.bdf_process = kratoscore.ComputeBDFCoefficientsProcess(model_part,2)
 
-        self.conv_criteria = incompressibleapp.VelPrCriteria(self.rel_vel_tol, self.abs_vel_tol,
-                                           self.rel_pres_tol, self.abs_pres_tol)
+        self.conv_criteria = cfd.VelPrCriteria(self.rel_vel_tol, self.abs_vel_tol,
```

Comment:
Note that this convergence criteria has been in the FluidDynamics (not the incompressible fluid) for a few years now... @rubenzorrilla I guess this is one more solver we should consider removing or updating at some point.

Reply:
Agree... Probably we can use the monolithic solver together with the stokes element.